recipes/pint-xarray/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "pint-xarray" %}
+{% set version = "0.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a48be363f2cd870c0c73e11b9e2b04160c1cfb089fee98277189a3afc26ecc45
+
+build:
+  number: 0
+  noarch: python
+  script: SETUPTOOLS_SCM_PRETEND_VERSION={{ version }} {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools >=42
+    - setuptools_scm >=3.4
+    - toml
+  run:
+    - importlib-metadata
+    - numpy >=1.17
+    - pint >=0.16
+    - python
+    - xarray >=0.16.1
+
+test:
+  imports:
+    - pint_xarray
+    - pint_xarray.tests
+  commands:
+    - pip check
+    - python -c 'from pint_xarray import __version__; assert __version__ != "0.0.0"'
```

Comment:
This is not really an important test, right? It is quite odd that you need to check the version in the package like this. Maybe it is some oddity in the setuptools_scn that I don't understand.

Reply:
this makes sure the build fails if `setuptools_scm` doesn't detect the version (see the comment on `SETUPTOOLS_SCM_PRETEND_VERSION`)

Reply:
I'm not an expert on setuptools_scm but that looks like a bad workaround for when your source is not on a VCS. I like the option `write_to` that writes a version file and avoids this. There is also the `pkg_resources` option, but that can be slow. Let's merge this but if you are connected with upstream I recommend taking a look into those options.